packages/date-picker/src/FilterDateRangePicker/components/Trigger/TriggerButtonBase/TriggerButtonBase.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import { render, screen } from "@testing-library/react"
+import { TriggerButtonBase, TriggerButtonBaseProps } from "."
+
+const TriggerButtonBaseWrapper = (props: Partial<TriggerButtonBaseProps>) => (
+  <TriggerButtonBase isOpen={false} {...props}>
+    Pancake
+  </TriggerButtonBase>
+)
+
+describe("<TriggerButtonBase />", () => {
+  it("has the required attributes when not expanded", () => {
+    render(<TriggerButtonBaseWrapper />)
+    const button = screen.getByRole("button", { name: "Open filter - Pancake" })
```

Comment:
@dougmacknz I had noticed that the button in FilterMultiSelect did not indicate whether clicking on the button was going to open or close the filter, so I added in something. I'm not sure if it's a good label or not though (prepending with `Open/Close filter`), so any advice?

In the actual case of the FilterDRP with a value, the label looks something like this: `Open filter - Dates: 1 May 2022 - 25 Nov 2022`, where `Dates` is the label and `1 May 2022 - 25 Nov 2022` is the selected value (which may or may not exist).

Reply:
`aria-expanded=true|false` should be communicating the open/closed state of the popup, so we shouldn't need to add it to the label.

https://user-images.githubusercontent.com/1811583/203704108-c17d2ac4-59fa-4703-94c2-088f0179d862.mov

Reply:
I see! Makes sense. Do we want to do anything about the label and the selected values being mushed together in the name (especially since it changes)? Or is that fine?

Reply:
<img width="347" alt="image" src="https://user-images.githubusercontent.com/1811583/203706788-fb587275-10bb-4a86-b985-82e8f3b06bf5.png">

Just gave this a quick test on VoiceOver and ran into something interesting: VoiceOver must have detected that it looks like a date range and tried to read out something more nice, except it screwed it up. It said "The 25th of November to 2030 November 2022"

So yeah it's probably a good idea for us to override the contents label with an aria-label that reads nicer.

Reply:
:(
Any suggestions?

Just tried out a few attempts with @mcwinter07 and any usage of `-` (including unicode) as it is causes the problem. It worked nicely when I gave it an aria-label of `"to"`, but then it means we'd need to translate it, and that may not translate well in other languages.
We also tried wrapping in `time` and `data` tags, but the same problem happens.
When we used a different separator, the dates were read out correctly, but the separator was not announced.

Would it still make sense if we just read the two dates? Or does that now cause confusion and no longer seems like a date range?

Reply:
Oh right, I didn't think about internationalisation. Maybe I'll test out the current label in NVDA (known for being fairly similar to JAWS) and we'll see how this goes. If it's just a VoiceOver bug, we may want to just roll with this for now, given the much lower number of people using VoiceOver.

Reply:
@mcwinter07 just tested on Airbnb and the same problem persists. It might be a VoiceOver bug after all?

Reply:
How about "Between {date} and {date}"

Reply:
Requires translation though 😞

Reply:
That can be handled in the solution for translating, as I understand a few other places aren't translated yet either?